client/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Graham's Search</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="shortcut icon" href="./img/favicon.ico" type="image/x-icon" />
+  </head>
+  <body>
+    <main class="wrapper">
+      <h1 class="title">Graham</h1>
```

Comment:
Also why Graham ? Also curious 😆 Graham crackers?

Also it's best not to use class title as there is an element called titled just for safety

Reply:
It was a (all be it not great) maths joke. google is from the word googol so we decided to call it Graham after another large number.

Good point about the title class, thank you

Reply:
Ah, no idea about that stuff but sounds awesome! And no problem!